modules/ROOT/pages/connected-apps-org-admin.adoc

```diff
@@ -53,6 +53,6 @@ Applications created by users in the same organization are marked as *internal a
 
 By default, connected apps are configured at the root organization level, not at the business group (sub-organization) levels.
 
-When you create a connected app to act on its own behalf using client credentials, you can choose which root organization or business groups to enable it in after you choose scopes.
+When you create a connected app to act on its own behalf using client credentials, you can choose which root organization or business groups to enable it in after you choose scopes. Apps that act on their own behalf are useful for CI/CD purposes since they do not require interaction from a user.
```

Comment:
```suggestion
When you create a connected app to act on its own behalf using client credentials, you can choose which root organization or business groups to enable it in after you choose scopes. Apps that act on their own behalf are useful for CI/CD purposes because they do not require interaction from a user.
```